output.txt

```diff
@@ -1,6 +1,41 @@
-------------+--------------------+--------+-----------+----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+---------------------+-------
-          1 | Running a Node App | coding | jessjelly | This is part two of a series on how to get up and running with Systemd and Node.js. This part dives deeper into how to successfully run your app with systemd long-term, and how to set it up in a production environment. | 2020-11-07 06:03:00 |     0
-(1 row)
+ article_id |                                         title                                         |  topic   |    author    |                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               body                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               |     created_at      | votes 
```

Comment:
try not to push these files to github!

only git add the correct files and/or add these files to your .gitignore!